pkg/measurements/routeadvertisement-latency_linux.go

```diff
@@ -747,6 +744,9 @@ func (r *raLatency) waitForSceanrioCompletion(desiredCount uint64, maxTimeout ti
 // Stop stops raLatency measurement
 func (r *raLatency) Stop() error {
 	var err error
+	if r.JobConfig.SkipIndexing {
```

Comment:
First, this doesn't seem related to this change.
Second, why is this handled here? Should it be handled in the caller? Also, Isn't this way to far in the process?